components/list/list-item-nav-button-mixin.js

```diff
@@ -72,6 +81,9 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 		if (this.current) {
 			this.dispatchSetChildCurrentEvent(true);
 		}
+
+		this.addEventListener('focusin', this.#stopPropagation);
```

Comment:
We only want focus styles (e.g., border) to apply to the item itself and not its parents. This prevents the parent from getting `_focusing` when the child is focused on. This would prevent a consumer from being able to listen for this event, so let me know concerns and I can try to figure out a different strategy.